### PR TITLE
updates/kajal-feb-2

### PR DESCRIPTION
**Pull request by kkaajjaall**

### CODE REVIEW

**Review by ithqindia — APPROVED** [review]

File: resources/views/admin/address.blade.php [hunk 1]
```diff
@@ -34,19 +36,6 @@ class="position-absolute translate-middle bottom-0 start-100 mb-6 bg-success rou
                                     <div class="d-flex align-items-center mb-2">
                                         <a href="#" class="text-gray-900 text-hover-primary fs-2 fw-bolder me-1">
```
Comment (ithqindia) [inline resources/views/admin/address.blade.php:37]: ` <a href="#"` change to `span`

File: resources/views/admin/counselor-session.blade.php [hunk 2]
```diff
@@ -3,31 +3,19 @@
...
+            @else
+                <img src="{{ asset('/img/blank-profile.png') }}" alt="{{ $user->name }}" class='border border-3'>
+            @endif
             </div>
         </div>
         <div class="flex-grow-1">
             <div class="d-flex justify-content-between aling-items-start flex-warp mb-2">
                 <div class="d-flex flex-column">
                     <div class="d-flex align-items-center mb-2">
                         <a class="text-gray-900 text-hover-primary fs-2 fw-b older me-1"> {{ $user->name }}</a>
```
Comment (ithqindia) [inline resources/views/admin/counselor-session.blade.php:18]: change to span

**Review by ithqindia — CHANGES_REQUESTED** [review]
<a class="text-gray-900 text-hover-primary fs-2 fw-b older me-1"> {{ $user->name }}</a>
change all to span tag

**Comment by kkaajjaall** [comment]
span added

**Review by ithqindia — APPROVED** [review]

**Comment by ithqindia** [comment]
Merged in https://github.com/ithqindia/metroclub/pull/158

**Pull request closed without merging**